File: exo3/exo3.php
<?php
//cette algo compte le nombre de a
$word = "jonathan";
$carac = "a";
$resultat = 0;
//initialisation du nombre de "a" a zero
$i = 0;
//initialisation de l'index donc index 0

while($i < strlen($word)){
//tant que index inferieur a la longueur du mot cela exectute le code ci-dessous
	if($word[$i] == $carac){
//si a l'index ou je me trouve c'est bien un "a" j'incremente de 1
		$resultat++;
	}
$i++;
//et je passe a l'index suivant
}

echo "le nombre de caractereRecherche est : " .$resultat."\r"."\n";

// "\r" =retour chariot et "\n" = passage a la ligne il faut bien utiliser les doubles qotes
?>

File: exo3/exo3.py
#cette algo compte le nombre de n

word = "anticonstitutionnement"
char = "n"
result = 0
#initialisation du nombre de "n" a zero
i = 0
#initialisation de l'index donc index 0

while i < len(word):
#tant que index inferieur a la longueur du mot cela exectute le code ci-dessous
	if word[i] == char:
#si a l'index ou je me trouve c'est bien un "n" j'incremente de 1
		result +=1
	i +=1
#et je passe a l'index suivant


print ("le nombre de caractereRecherche est : " + str(result))
